Azure/azure-cli
# [RDBMS] `az mysql flexible-server identity remove`: Allow removing all identities in a MySQL server

**Related command**
`az mysql flexible-server identity remove`

**Description**
When customer wants to remove all the user managed identities in a MySQL flexible server, RP will throw an error saying that we can't do that. Instead, when trying to remove all existing identities, we should send only `{"identity": {"type": "None"}}`.

**Testing Guide**
- Assign two identities to an existing server with no identities: `az mysql flexible-server identity assign -g resourceGroup -s serverName --identity testIdentity1 testIdentity2`
- Remove the first identity: `az mysql flexible-server identity remove -g resourceGroup -s serverName --identity testIdentity1`
- Remove the second identity (this now fails but this PR fixes it): `az mysql flexible-server identity remove -g resourceGroup -s serverName --identity testIdentity2`
- Verify that server doesn't have identities: `az mysql flexible-server identity list`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
